src/math/spacegroup.cpp

```diff
@@ -230,11 +234,18 @@ namespace OpenBabel
                             t = &v.z();
                             break;
                           }
+                        // Cover cases like 7/6
+                        double numerator = (double) (row[j] - '0');
+                        double denumerator = (double) (row[j+2] - '0');
+                        if (numerator > denumerator)
+                          numerator -= ((int)numerator % (int)denumerator) * denumerator;
+
                         *t = ((double) (row[j] - '0')) / (row[j+2] - '0');
                         if (neg)
                           *t = - *t;
+
+                        j +=2;
                       }
-                    j +=2;
```

Comment:
If we have an integer which is not followed by '/', it is hopefully this situation:
`2-x,1-y,1-z`
where the correct thing is to remove the integers to:
`-x,-y,-z`